vocode/streaming/streaming_conversation.py

```diff
@@ -1513,7 +1497,7 @@ async def send_speech_to_output(
         if (
             not stop_event.is_set()
             and not buffer_cleared
-            and not message_sent in message_options
+            and message_sent not in message_options
```

Comment:
is there a difference between `not x in list` and `x not in list`? just curious

Reply:
No my formatter/linter automatically makes those changes to the file everytime i save it. It's because of pep8 I think